Lib/asyncio/locks.py

```diff
@@ -181,18 +181,22 @@ def locked(self):
             self._locked = True
             return True
 
+        _waiters = self._waiters
```

Comment:
Nit: just use `self._waiters`. No need for a new variable.

Reply:
New variable was created in order to avoid the double attribute lookup, I'll remove it if you think it's more readable though.